delivery/model.py

```diff
@@ -26,25 +30,37 @@ class GithubUser:
     github_hostname: str
 
 
-@dataclasses.dataclass(frozen=True) # TODO: deduplicate w/ modelclass in delivery-service/yp.py
-class Sprint:
-    name: str
-    end_date: datetime.datetime
-    release_decision: datetime.datetime
-    rtc: datetime.datetime
-    canary_freeze: datetime.datetime
+@dataclasses.dataclass(frozen=True)
+class SprintEvent:
+    value: datetime.datetime | str # e.g. <iso-timestamp> or <sprint-name>
+    attribute: str # e.g. "rtc"
+    attribute_display_name: str # e.g. "Release To Customer"
 
     @staticmethod
     def from_dict(raw: dict):
         return dacite.from_dict(
-            data_class=Sprint,
+            data_class=SprintEvent,
```

Comment:
naming: not every data describing a Sprint is actually an event. So let us not choose a name implying that.

we have:
- name (obviously not an event, is it?)
- dates (start, end, rtc. ..)

Considering that there seem to be two different attributes that make up a sprint (name and dates), it might be worthwhile reflecting that in the model, maybe like so:

```
# an example sprint in yaml form
- name: 2208a # or: id?
  dates: # or: data / attributes?
  - name: end_date # name is a technical id which we will keep stable
    display_name: End Date
    value: 2023-09-09
  # ...
- name: 2208b
  # ...
```

Rationale: Sprints will for sure always have a name, so it might be more adequate to express this explicitly in the model, rather than making the model so generic that even that could be subject to customisation.